docs/platforms/platforms_index.rst

```diff
@@ -10,7 +10,7 @@ Across allocated compute nodes, libEnsemble's flexible architecture lends itself
 best to two general modes of worker distributions. The first mode we refer
 to as **centralized** mode, where the libEnsemble manager and worker processes
 are grouped on one or more nodes but through the libEnsemble executor or a
-job-launch command can execute calculations on the other allocated nodes:
+task-launch command can execute calculations on the other allocated nodes:
```

Comment:
I think I prefer "job" here because the context now refers to commands like `srun` or `qsub`, job-launch commands.

Reply:
I like the change to `srun`/`qsub`. Let's be specific, when possible.